ts-eks/src/userInfo.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * @klotho::execution_unit {
+ *  id = "sequelize-main"
+ * }
+ */
+
+import {
+  DataTypes,
+  InferAttributes,
+  InferCreationAttributes,
+  Model,
+  Sequelize,
+} from "sequelize";
+import {getUsers} from './users'
+
+const sequelize = setupSequelize();
+
+async function setupSequelize(): Promise<Sequelize> {
+
+  /** @klotho::persist  {
+   *   id = "sequelizeDB"
+   * }
+   * */ 
+  const sequelize = new Sequelize(`sqlite::memory:`, { logging: false });
```

Comment:
double checking if we need to have sequelize as part of this sample. I thought it was the ts-microservices sample initially. 

But yeh more so to take into consideration the spin up times and all that.

Reply:
right, i just want to test the env vars for eks which has to be either RDS or elasticache, both take a while. I think its worth the tests even thought the timing stinks